README.md

```diff
@@ -18,26 +18,27 @@ things MLOps.
 ## Why BentoML? ##
 
 🍱 Easily go from training to model serving in production
-- Support multiple ML frameworks natively: Tensorflow, PyTorch, XGBoost, Scikit-Learn and many more!
-- Define custom serving pipeline with pre-processing, post-processing and ensemble models
+- Supports multiple ML frameworks natively: Tensorflow, PyTorch, XGBoost, Scikit-Learn and many more!
+- Defines custom serving pipeline with pre-processing, post-processing and ensemble models
+- Enables multi-model inference graph with model runners architecture
 - Standard `.bento` format for packaging code, models and dependencies for easy versioning and deployment
-- Integrate with any training pipeline or ML experimentation platform
+- Integrates with any training pipeline or ML experimentation platform
 
 ✨ Model Serving the way you need it 
 - **Online serving** via REST API or gRPC
 - **Offline scoring** on batch datasets with Apache Spark, or Dask
 - **Stream serving** with Kafka, Beam, and Flink
 
 🚢 Deployment workflow made for production
-- 🐳 Automatically generate docker images for production deployment
+- 🐳 Automatically generates docker images for production deployment
 - [🦄️ Yatai](https://github.com/bentoml/yatai): Model Deployment at scale on Kubernetes
 - [🚀 bentoctl](https://github.com/bentoml/bentoctl): Fast model deployment on any cloud platform
 
 🐍 Python-first, scales with powerful optimizations
-- Parallelize compute-intense model inference workloads to scale separately from the serving logic
+- Parallelizes compute-intense model inference workloads to scale separately from the serving logic
 - Adaptive batching dynamically groups inference requests for optimal performance
-- Orchestrate distributed inference graph with multiple models via Yatai on Kubernetes
-- Easily configure CUDA dependencies for running inference with GPU
+- Orchestrates distributed inference graph with multiple models via Yatai on Kubernetes
+- Easily configures CUDA dependencies for running inference with GPU
```

Comment:
```suggestion
- Easily configure CUDA dependencies for running inference with GPU
```

I think the user is doing the configuring here. It does feel out of place like this, though.

Reply:
Actually, how about:

```suggestion
- Allows for easy configuration of CUDA dependencies for running inference with GPU
```

Reply:
How about
```suggestion
- Enables CUDA dependencies customization for running inference with GPU with ease!
```

Reply:
I think that wordings a bit strange because it implies that users weren't able to do this before :P

Reply:
which is true right 😆